dll/win32/shlwapi/ordinal.c

```diff
@@ -4852,7 +4866,11 @@ INT WINAPIV ShellMessageBoxWrapW(HINSTANCE hInstance, HWND hWnd, LPCWSTR lpText,
 
     __ms_va_end(args);
 
+#ifdef __REACTOS__
     ret = MessageBoxW(hWnd, pszTemp, pszTitle, uType);
+#else
+    ret = MessageBoxW(hWnd, pszTemp, pszTitle, uType | MB_SETFOREGROUND);
```

Comment:
See comment above regarding the setforeground flag.

Reply:
nit: you can add:
```
#ifdef __REACTOS__
uType |= MB_SETFOREGROUND
#endif
```
and reduce the diff.